### DISCUSSION TITLE
MB you should stop supporting this freaky lib?

### DISCUSSION THREAD

**Discussion by AndreyStille** [huggingface/diffusers#10408] [Ideas]
I hate this lib with all my heart and soul. You are just killing open-source with this monopoly, freaky lib which provide light access for amateurs and provide ass ache for those who just want to try some open source models. 
Can you just stop? If you have something good in your hearts.
How easy it was to test models at the time when weights and architectures weren't combined in this syphilitic, ugly tool

**Comment by asomoza** [comment]
closing this as this is just a rant and by an user that only uses the account for this type of posts.

Evidence: [issue1](https://github.com/tensorflow/tensorflow/issues/62918) and [issue2](https://github.com/tensorflow/tensorflow/issues/74695)
